Sources/Internal/SnapshotStructure.swift

```diff
@@ -3,11 +3,13 @@ import DifferenceKit
 
 struct SnapshotStructure<SectionID: Hashable, ItemID: Hashable> {
     struct Item: Differentiable, Equatable {
-        var differenceIdentifier: ItemID
+        var differenceIdentifier: Int
+        var identifier: ItemID
         var isReloaded: Bool
 
         init(id: ItemID, isReloaded: Bool) {
-            self.differenceIdentifier = id
+            self.identifier = id
+            self.differenceIdentifier = id.hashValue
```

Comment:
Hash value is deprecated, so maybe do
```suggestion
            var hasher = Hasher()
            hasher.combine(id)
            self.differenceIdentifier = hasher.finalize()
```

Reply:
Actually, the [docs](https://developer.apple.com/documentation/swift/hashable/1540917-hashvalue) state that "hashValue is deprecated as a Hashable requirement." However, the variable itself is not deprecated, and is intended as a way to get the exact result produced by the three lines of code you provided.

Reply:
Ok, was not sure that it was meant like that 😅